js/main.js

```diff
@@ -20,8 +21,76 @@ function saveButtonHandler(event) {
   data.entries.unshift(entriesObj);
   img.setAttribute('src', 'images/placeholder-image-square.jpg');
   form.reset();
+  entriesContainer.className = 'container';
+  formContainer.className = 'container hidden';
+  ul.prepend(addEntry(entriesObj));
+  localStorage.setItem('current', 'entries-current');
```

Comment:
What is the purpose of this line? You should be saving the current view into your data model, rather than directly in localStorage